src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityTreeBrowser.java

```diff
@@ -246,7 +227,8 @@ public void clearRecordsFetchedChildren() {
 	 * Private Methods
 	 */
 	private void fireEntitySelectedEvent() {
-		handlerManager.fireEvent(new EntitySelectedEvent());
```

Comment:
This should be a place change, rather than communicating this back up to the entitypagetop